### PR TITLE
Fix/webpack5

### PR DESCRIPTION
**Pull request by skjnldsv**
Fixed a few errors and dependencies

| Before | After |
|:---------|:------|
|267 passing (1m)<br>11 pending<br>38 failing|272 passing (1m)<br>11 pending<br>33 failing|

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "format": "prettier '{src,test}/**/*' ./*.json --write --loglevel warn",
     "lint": "eslint src/**/*.ts test/**/*.ts --fix",
     "test": "yarn run clean-tmp && yarn run build && ts-mocha --timeout 10000 --recursive --require @babel/register --exit \"{test,src}/**/*.test.ts\"",
+    "test:file": "yarn run clean-tmp && yarn run build && ts-mocha --timeout 10000 --recursive --require @babel/register --exit",
```
Comment (jayaddison) [inline package.json:23]: What's the `test:file` script used for @skjnldsv?

**Comment by jayaddison** [comment]
This looks very helpful, thanks!  Moving to the built-in Webpack dev server looks sensible, amongst the other fixups here.

Do you want to mark this as ready-for-review so that I could merge it into https://github.com/sysgears/mochapack/pull/83?  If you're fixing up / developing things a bit further then that's fine too.

File: package.json (hunk 1 above)
Comment (skjnldsv) [reply package.json:23]: Ah it's a leftover. You can specify path to a unique test file. It's pretty useful to not run the entire test suite but only one.
You can remove if you want :)

**Comment by skjnldsv** [comment]
@jayaddison feel free to merge

**Pull request merged**